src/popup/components/FungibleTokens/TokenDetailsInfo.vue

```diff
@@ -0,0 +1,263 @@
+          }"
+        >
+          {{ Number(tokenData.market_cap_change_percentage_24h).toFixed(2) }}%
+        </span>
+        {{ formatCurrency(tokenData.market_cap_change_24h) }}
+      </template>
+    </DetailsRow>
+    <DetailsRow
+      v-if="!tokenData.isAe && UNFINISHED_FEATURES"
+      :label="$t('pages.token-details.feeDaily')"
+    />
+
+    <DetailsRow
+      v-if="!tokenData.isAe"
+      class="link"
+      :label="$t('pages.token-details.chart')"
+    >
+      <template #text>
+        <a
+          :href="DEX_URL"
+          target="_blank"
+        >
+          {{ displayDexUrl }}
+          <ExternalLink />
+        </a>
+      </template>
+    </DetailsRow>
+    <DetailsRow
+      v-if="!tokenData.isAe && UNFINISHED_FEATURES"
+      :label="$t('pages.token-details.price-ae')"
+    />
+    <DetailsRow
+      v-if="tokenData.current_price"
+      class="price"
+      :label="$t('pages.token-details.price')"
+    >
+      <template #text>
+        <span
+          :class="{
+            green: tokenData.price_change_percentage_24h > 0,
+            red: tokenData.price_change_percentage_24h < 0,
+          }"
+        >
+          {{ Number(tokenData.price_change_percentage_24h).toFixed(2) }}%
+        </span>
+        {{ formatCurrency(tokenData.current_price) }}
+      </template>
+    </DetailsRow>
+    <DetailsRow
+      v-if="tokenData.ath"
+      :label="$t('pages.token-details.ath-change')"
+      :text="formatCurrency(tokenData.ath)"
+    />
+    <DetailsRow
+      v-if="tokenData.atl"
+      :label="$t('pages.token-details.atl-change')"
+      :text="formatCurrency(tokenData.atl)"
+    />
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+import BigNumber from 'bignumber.js';
+import DetailsRow from './DetailsRow.vue';
+import AddressShortening from '../AddressShortening.vue';
+import Tokens from '../Tokens.vue';
+import ExternalLink from '../../../icons/external-link.svg?vue-component';
+import { DEX_URL } from '../../utils/constants';
+import { convertToken } from '../../utils/helper';
+
+export default {
+  components: {
+    DetailsRow,
+    AddressShortening,
+    ExternalLink,
+    Tokens,
+  },
+  props: {
+    tokenData: {
+      type: Object,
+      required: true,
+    },
+    tokenPairs: {
+      type: Object,
+      required: true,
+    },
+    tokens: {
+      type: Array,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      UNFINISHED_FEATURES: process.env.UNFINISHED_FEATURES,
+      DEX_URL,
+    };
+  },
+  computed: {
+    ...mapGetters(['formatCurrency', 'formatNumber']),
+    poolShare() {
+      if (!this.tokenPairs || !this.tokenPairs.balance || !this.tokenPairs.totalSupply) {
+        return null;
+      }
+      return `${BigNumber(this.tokenPairs.balance)
+        .times(100).div(this.tokenPairs.totalSupply).toFixed(8)}%`;
+    },
+    displayDexUrl() {
+      return this.DEX_URL.replace('https://', '');
```

Comment:
`DEX_URL` is imported above so I suggest to not use the `this.` as it's not necessary.